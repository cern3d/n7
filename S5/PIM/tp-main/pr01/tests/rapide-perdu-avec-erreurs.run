./allumettes <<EOF
r
n
-2
0
4
1
1
3
2
1
1
1
1
1
1
1
1
1
1
1
1
1
1
EOF
