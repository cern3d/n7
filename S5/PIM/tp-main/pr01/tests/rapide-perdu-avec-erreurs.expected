Niveau de l'ordinateur (n)aïf, (d)istrait, (r)apide ou (e)xpert ? Mon niveau est rapide.
Est-ce que vous commencez (o/n) ? 
| | | | |   | | | | |   | | |
| | | | |   | | | | |   | | |
| | | | |   | | | | |   | | |

Je prends 3 allumettes.

| | | | |   | | | | |
| | | | |   | | | | |
| | | | |   | | | | |

Combien d'allumettes prenez-vous ? Arbitre : Il faut prendre au moins une allumette.
Combien d'allumettes prenez-vous ? Arbitre : Il faut prendre au moins une allumette.
Combien d'allumettes prenez-vous ? Arbitre : Il est interdit de prendre plus de 3 allumettes.
Combien d'allumettes prenez-vous ? 
| | | | |   | | | |
| | | | |   | | | |
| | | | |   | | | |

Je prends 3 allumettes.

| | | | |   |
| | | | |   |
| | | | |   |

Combien d'allumettes prenez-vous ? 
| | | | |
| | | | |
| | | | |

Je prends 3 allumettes.

| |
| |
| |

Combien d'allumettes prenez-vous ? Arbitre : Il reste seulement 2 allumettes.
Combien d'allumettes prenez-vous ? 
J'ai gagné.
